airflow/models/connection.py

```diff
@@ -18,6 +18,7 @@
 
 import json
 import warnings
+import logging
```

Comment:
This won’t pass linters. Please run pre-commit locally and fix the errors first.